src/layman/layer/qgis/wms.py

```diff
@@ -76,6 +78,11 @@ def save_qgs_file(workspace, layer):
                                          layer_bbox, source_type)
     with open(get_layer_file_path(workspace, layer), "w") as qgs_file:
         print(qgs_str, file=qgs_file)
+    external_images_original_path = layer_info.get('_style', dict()).get('external_images_dir')
+    if external_images_original_path:
+        external_images_dir_name = os.path.basename(external_images_original_path)
+        external_images_target_path = os.path.join(qgis.get_layer_dir(workspace, layer), external_images_dir_name)
+        copy_tree(external_images_original_path, external_images_target_path)
 
```

Comment:
Add layer with QML and external images to static_data and test at least thumbnail